oslo/torch/nn/modules/linear.py

```diff
@@ -396,17 +398,27 @@ def __init__(
         super().__init__(
             in_features=int(in_features // self.tesseract_dim),
             out_features=int(out_features // self.tesseract_dim),
-            bias=bias,
+            bias=False,
```

Comment:
### 
```suggestion
            bias=bias,
```

Reply:
@jason9693 why is this `False` ?

Reply:
@hyunwoongko you can see the reason at the my comment right above.